mivs/model_checks.py

```diff
@@ -18,7 +18,7 @@ def _is_invalid_url(url):
 
 @validation.IndieStudio
 def new_studio_deadline(studio):
-    if studio.is_new and c.AFTER_ROUND_ONE_DEADLINE:
+    if studio.is_new and c.AFTER_ROUND_ONE_DEADLINE and c.INDIE_ADMIN not in AdminAccount.access_set():
```

Comment:
I always forget about that :)   I switched all of it to use c.HAS_INDIE_ADMIN_ACCESS, makes other parts simpler too.